### PR TITLE
Updated statuses

### PR DESCRIPTION
**Pull request by Qwerty-Space**
#hacktoberfest did I do this right?

### CODE REVIEW

**Comment by Qwerty-Space** [comment]
Sorry for the faff

File: long-live-pomf.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 |                            |                  | https://github.com/kimoi/madokami.com         |                  |
 |                            |                  | https://github.com/bohrmeista/1338            |                  |
 |                            |                  | https://github.com/nya/cpomf                  |                  |
-| https://nup.pw/            | Up               |                                               |              150 | JS-free
+| https://nup.pw/            | Down (01/Oct-17) |                                               |              150 | JS-free
```
Comment (tsdko) [inline long-live-pomf.md]: You used to use `/` instead of `-`, why the change?

File: long-live-pomf.md [hunk 2]
```diff
@@ -26,16 +26,16 @@
 | https://sugoi.vidyagam.es/ | Up               |                                               |              100 | Nice colors, pastebin
 | https://vidga.me/          | Up               |                                               |              100 | JS-free
 | https://yiff.moe/          | Down             |                                               |              512 | Nice colors, metadata stripping
-| https://pomf.sinister.ly/  | Up               |                                               |              100 |
+| https://pomf.sinister.ly/  | Down (01/Oct-17) |                                               |              100 |
```
Comment (tsdko) [inline long-live-pomf.md]: Works with http

File: long-live-pomf.md (hunk 1 above)
Comment (Qwerty-Space) [reply long-live-pomf.md]: A-accident?

File: long-live-pomf.md [hunk 3]
```diff
@@ -26,16 +26,16 @@
 | https://sugoi.vidyagam.es/ | Up               |                                               |              100 | Nice colors, pastebin
 | https://vidga.me/          | Up               |                                               |              100 | JS-free
 | https://yiff.moe/          | Down             |                                               |              512 | Nice colors, metadata stripping
-| https://pomf.sinister.ly/  | Up               |                                               |              100 |
+| http://pomf.sinister.ly/   | Up               |                                               |              100 | Not secure - http only
```
Comment (tsdko) [inline long-live-pomf.md]: `Not secure` is redundant, as it applies to all http sites. Keep in mind that http entries are supposed to be below all https ones.
